src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -29,21 +32,30 @@
     private final String email;
     private final String address;
     private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final String studentId;
+    private final String tutorialName;
+
 
-    /**
-     * Constructs a {@code JsonAdaptedPerson} with the given person details.
-     */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
+            @JsonProperty("studentId") String studentId,
+            @JsonProperty("tutorialName") String tutorialName) {
         this.name = name;
         this.phone = phone;
         this.email = email;
         this.address = address;
         if (tagged != null) {
             this.tagged.addAll(tagged);
         }
+        if (studentId == null && tutorialName == null) {
```

Comment:
i think we should check studentId == null || tutorialName == null, as anything without either of these fields is not considered a valid student

Reply:
Right! I will do the changes

Reply:
> i think we should check studentId == null || tutorialName == null, as anything without either of these fields is not considered a valid student
Right! I will do the changes